packages/components/src/mobile/color-settings/utils.native.js

```diff
@@ -1,11 +1,16 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
 const gradients = {
 	linear: 'linear-gradient',
 	radial: 'radial-gradient',
 };
 
 const gradientOptions = [
-	{ label: 'Linear', value: gradients.linear },
-	{ label: 'Radial', value: gradients.radial },
+	{ label: __( 'Linear' ), value: gradients.linear },
+	{ label: __( 'Radial' ), value: gradients.radial },
```

Comment:
`gradientOptions` is being used in Custom Gradient Picker component:
https://github.com/WordPress/gutenberg/blob/3da717b8d0ac7d7821fc6d0475695ccf3ae2829f/packages/components/src/custom-gradient-picker/index.native.js#L31
https://github.com/WordPress/gutenberg/blob/3da717b8d0ac7d7821fc6d0475695ccf3ae2829f/packages/components/src/custom-gradient-picker/index.native.js#L94-L97